index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <title>Presto | Documentation</title>
```

Comment:
Unrelated to this commit. It should be in a separate commit.

Reply:
Ok